app/javascript/App/bundles/NewsfeedPage/NewsListItem.tsx

```diff
@@ -44,9 +45,20 @@ const NewsListItem = (props) => {
     .replace(/<h1>/g, '')
     .replace(/<\/h1>/g, '')
 
+
+  let titleStyle: React.CSSProperties  = {};
+  if (hasRead) {
+    titleStyle.color = '#999';
+  }
+
+  let cn = className;
+  if (isUnseen) {
+    cn += ' unseen';
+  }
+
   return (
     <div
-      className={className}
+      className={cn}
```

Comment:
as a rule of thumb, try to avoid one and two letter variable names

Reply:
👍